ArduPlane/quadplane.cpp

```diff
@@ -2910,24 +2920,50 @@ void QuadPlane::Log_Write_QControl_Tuning()
   reduces the need for down pitch which reduces load on the vertical
   lift motors.
  */
-int8_t QuadPlane::forward_throttle_pct(void)
-{
+int8_t QuadPlane::forward_throttle_pct(bool tiltrotor)
+{
+    // if not a tiltrotor, disable forward motor:
+    if (!tiltrotor) {
```

Comment:
I think we should be able to make this work for all quad planes

Reply:
That comment may be confusing... manual forward throttle does work for standard quadplanes (SITL tested on the RF8 BigStik quadplane), just not when disarmed.

The change in behavior here is only that tiltrotor motors can tilt forward when disarmed.  This is a good thing because you don't want them to suddenly tilt forward on arming since some planes (like the Convergence) have no clearance to the ground.  
This method is only called with tiltrotor==true from tiltrotor_continuous_update, and that method only affects tilt servos, not throttles.

For a non-tiltrotor, we must return zero here when disarmed or spooled down, otherwise props could spin when they shouldn't.